MaxText/llama_or_mistral_ckpt.py

```diff
@@ -532,12 +555,18 @@ def checkpoint_device_put(arr):
   parser.add_argument("--base-model-path", type=str, required=True)
   parser.add_argument("--maxtext-model-path", type=str, required=True)
   parser.add_argument("--model-size", type=str, required=True)
+  parser.add_argument("--checkpoint-type", type=str, required=True)
```

Comment:
If this is required, we will need to other ckpt tests to add a flag `-checkpoint-type=pth`.